css/languages.css

```diff
@@ -1,33 +1,48 @@
-
 .language-main-container {
   display: grid;
-  grid-template-columns: 327px;
-  grid-template-rows: repeat(3, 352px);
+  grid-template-rows: repeat(3, 1fr);
+  grid-template-columns: 1fr;
   row-gap: 24px;
-  justify-content: center;
+  margin: 0 26px;
+  background-color: #fff;
 }
 
 .language-sub-container {
-  border: 1px solid #c1c7d0;
   background: #ebf0ee;
+  border: 1px solid #c1c7d0;
   border-radius: 8px;
-  padding-top: 30px;
+}
```

Comment:
[OPTIONAL]  Currently your design is not matching the Figma template on bigger screens consider reviewing it to make it more professional.

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/221551472-e88ae78a-0b23-47e7-9ba8-0577edb7764f.png)